src/utils.jl

```diff
@@ -370,50 +370,83 @@ function codelocation(code::CodeInfo, idx::Int)
     return codeloc
 end
 
-function compute_corrected_linerange(method::Method)
-    _, line1 = whereis(method)
-    offset = line1 - method.line
-    src = JuliaInterpreter.get_source(method)
-    lastline = linetable(src)[end]::LineTypes
-    return line1:getline(lastline) + offset
+function compute_corrected_linerange(framecode)
+    scope = framecode.scope
+    if scope isa Method
+        method = framecode.scope
+        _, line1 = whereis(method)
+        offset = line1 - method.line
+        src = JuliaInterpreter.get_source(method)
+        lastline = linetable(src)[end]::LineTypes
+        return line1:getline(lastline) + offset
+    else
+        getline(linetable(framecode, 1)):getline(last(linetable(framecode)))
+    end
 end
```

Comment:
Can we make this multiple dispatch instead? We have some code in DebugAdapter that calls `compute_corrected_linerange` with a method argument.

Reply:
Should be back now.